rocAL/rocAL/source/augmentations/geometry_augmentations/node_random_crop.cpp

```diff
@@ -57,17 +52,38 @@ void RandomCropNode::update_node()
     std::vector<uint32_t> crop_h_dims, crop_w_dims;
     _crop_param->get_crop_dimensions(crop_w_dims, crop_h_dims);
     _outputs[0]->update_tensor_roi(crop_w_dims, crop_h_dims);
+
+    // Obtain the crop coordinates and update the roi
+    auto x1 = _crop_param->get_x1_arr_val();
+    auto y1 = _crop_param->get_y1_arr_val();
+    RocalROI *crop_dims = static_cast<RocalROI *>_crop_coordinates;
+    for(unsigned i = 0; i < _batch_size; i++) {
+        crop_dims[i].x1 = x1[i];
+        crop_dims[i].y1 = y1[i];
+        crop_dims[i].x2 = crop_w_dims[i];
+        crop_dims[i].y2 = crop_h_dims[i];
+    }
 }
 
-void RandomCropNode::init(float crop_area_factor, float crop_aspect_ratio, float x_drift, float y_drift)
-{
+void RandomCropNode::init(float crop_area_factor, float crop_aspect_ratio, float x_drift, float y_drift) { }    // Is this required?
```

Comment:
Reccheck this comment if this init() is required. & remove the comment once verified